packages/react/src/components/selectable/selectable.react-template.tsx

```diff
@@ -40,7 +42,7 @@ export const reactSelectable: ComponentImplementation<Selectable<JSX.Element>> =
           slot={slot}
           onDsoChange={(e: CustomEvent<Event>) => dsoChange?.(e.detail)}
         >
-          {label}
+          {!labelledById ? label : ""}
```

Comment:
Deze gewoon renderen als label is opgegegeven

Reply:
Goed gezien